src/MLJTSVDInterface.jl

```diff
@@ -19,15 +19,12 @@ decomposition (SVD). Contrary to PCA, this estimator does not center the data be
 the singular value decomposition. This means it can work with sparse matrices efficiently.
 
 """
-mutable struct TSVDTransformer{R <: AbstractRNG} <: MLJModelInterface.Unsupervised
-    nvals::Int
-    maxiter::Int
-    rng::R
+MMI.@mlj_model mutable struct TSVDTransformer <: MLJModelInterface.Unsupervised
+    nvals::Int = 2
+    maxiter::Int = 1000
+    rng::Int = 123
```

Comment:
No, sorry, I should have been more explicit. I had rather in mind `Union{Integer,AbstractRNG}`. See for example [here](rng::Union{Int,AbstractRNG}) for an example of typing, and [here](https://github.com/JuliaAI/MLJEnsembles.jl/blob/7ca66d70a67cc7d9844e4a1b3365bf5b449b5cc0/src/ensembles.jl#L463) for an example of pre-processing in `fit`.

Reply:
no worries - the most recent change should reflect this!